projects/novo-elements/src/utils/form-utils/FormUtils.ts

```diff
@@ -795,4 +810,21 @@ export class FormUtils {
       return startDate;
     }
   }
+
+  inflateEmbeddedProperties(data: object): object {
+    if (data) {
+      Object.keys(data)
+      .filter((fieldName) => fieldName.includes('.'))
+      .forEach((field) => {
+          let [parentFieldName, fieldName] = field.split('.');
+          if (!data[parentFieldName]) {
+              data[parentFieldName] = {};
+          }
+          data[parentFieldName][fieldName] = data[field];
+          delete data[field];
+      });
+    }
+
+    return data;
```

Comment:
is this function being called by anything?

Reply:
can you please remove `examples.module.ts` from your commit? this is auto generated. we'll probably want to add this to our .gitignore as well.

Reply:
Yes, the method is used for embedded and inline_embedded fields. I have removed examples.module.ts and added it to the gitignore file.